Development/nmos/server_utils.cpp

```diff
@@ -76,6 +95,11 @@ namespace nmos
                     {
                         ctx.use_tmp_dh(boost::asio::buffer(dh_param.data(), dh_param.size()));
                     }
+
+#if !defined(_WIN32) || defined(CPPREST_FORCE_HTTP_CLIENT_ASIO)
```

Comment:
Unnecessary `#if`...`#endif` here because the whole function is already inside the same?